package.json

```diff
@@ -162,7 +163,7 @@
     "test-unit": "node test/pretest.js && mocha",
     "test-admin": "mocha --opts test/mocha-admin.opts",
     "test-e2e": "node test/e2e/server.js --env default",
-    "test-e2e-bg": "export KNE_SELENIUM_START_PROCESS=false && node test/e2e/server.js --env default",
+    "test-e2e-bg": "cross-env KNE_SELENIUM_START_PROCESS=false && node test/e2e/server.js --env default",
```

Comment:
Hmmm @webteckie I might be missing something, but does setting the environment variable like this actually do anything..? It looks to me like KNE overwrites it? 

https://github.com/keystonejs/keystone-nightwatch-e2e/blob/master/index.js#L101